op_crates/fetch/lib.rs

```diff
@@ -433,5 +434,5 @@ fn create_http_client(
   }
   builder
     .build()
-    .map_err(|_| deno_core::error::generic_error("Unable to build http client"))
+    .map_err(|e| generic_error(format!("Unable to build http client: {}", e)))
```

Comment:
Future stuff: we should consider a different error type here before stabilizing.

Reply:
Noted!